config/config.yaml

```diff
@@ -8,6 +8,8 @@ kraken:
             - -    scenarios/etcd.yml
             - -    scenarios/regex_openshift_pod_kill.yml
               -    scenarios/post_action_regex.py
+            - -    scenarios/customapp_pod.yml
+              -    scenarios/post_action_customapp_pod.py
```

Comment:
@juliemathew we can definitely have the scenario part of the scenarios to consume in the config but we might want to delete the entry in the default config since the acme-air namespace might not exist on a default cluster. Thoughts?

Reply:
@chaitanyaenr 
I agree to your suggestion. Removed the entry in the config file and modified the ReadMe for 'Pod Scenarios'.
Please review. Thanks.